resources/github-actions.md

```diff
@@ -125,6 +125,8 @@ As an example, `rapids-print-env` is used to print common environment informatio
 
 CI artifacts are uploaded to the GitHub Actions artifact store.
 
+Artifacts produced in public repositories can generally be accessed by anyone, using the methods described below.[^1]
```

Comment:
I'm happy with how this is rendering in the preview

ref: https://deploy-preview-609--docs-rapids-ai.netlify.app/resources/github-actions/#downloading-ci-artifacts

<img width="1404" alt="Screenshot 2025-05-30 at 3 07 49 PM" src="https://github.com/user-attachments/assets/87d7bf8f-0699-4f91-adbd-34c94b37a7c4" />

<img width="1448" alt="Screenshot 2025-05-30 at 3 10 47 PM" src="https://github.com/user-attachments/assets/6161136c-4e6d-4091-abd4-23aba3cd39c6" />